lightkurve/targetpixelfile.py

```diff
@@ -350,19 +363,25 @@ def plot(self, frame=0, cadenceno=None, bkg=False, aperture_mask=None,
         except IndexError:
             raise ValueError("frame {} is out of bounds, must be in the range "
                              "0-{}.".format(frame, self.flux.shape[0]))
-        fig, ax = plot_image(pflux, title='Kepler ID: {}'.format(self.keplerid),
+        if ax is None:
+            ax = plot_image(pflux, title='Kepler ID: {}'.format(self.keplerid),
                              extent=(self.column, self.column + self.shape[2],
-                             self.row, self.row + self.shape[1]), **kwargs)
-
+                             self.row, self.row + self.shape[1]), colorbar=colorbar,
+                             **kwargs)
+        else:
+            ax = plot_image(pflux, ax=ax, title='Kepler ID: {}'.format(self.keplerid),
+                             extent=(self.column, self.column + self.shape[2],
+                             self.row, self.row + self.shape[1]), colorbar=colorbar,
+                             **kwargs)
```

Comment:
Looks like the following code is sufficient here (?)
```
ax = plot_image(pflux, ax=ax, title='Kepler ID: {}'.format(self.keplerid),
                extent=(self.column, self.column + self.shape[2], self.row, self.row + self.shape[1]),
                colorbar=colorbar, **kwargs)
```

Reply:
Yas